python/README.md

```diff
@@ -13,9 +11,35 @@ The images below include variations for only the core packages in the [Intel® D
 | `3.10-full`, `latest`  | `2024.2.0` |
 | `3.10-core`            | `2024.2.0` |
 
-## Build from Source
+## Run a Performance Sample
+
+To run a performance sample run the following commands:
+
+```bash
+git clone https://github.com/intel/ai-containers
+cd ai-containers/python
+docker run --rm -it \
+    -v $PWD/tests:/tests \
+    intel/python:latest \
+    python /tests/perf_sample.py
+```
+
+### Compare the results against stock python
 
-To build the images from source, clone the [Intel® AI Containers](https://github.com/intel/ai-containers) repository, follow the main `README.md` file to setup your environment, and run the following command:
+In the previous command, you should see a result at the bottom like: `Time Consuming: 0.03897857666015625`. We can compare this against `python:3.11-slim-bullseye`
+
```

Comment:
what directory user should be in before running this command? need to mention that

Reply:
I added a note regarding that